train_segmentation_model.py

```diff
@@ -91,35 +92,26 @@ def train(gcp_bucket, config_file):
         train_config['loss'],
         train_config['optimizer'])
 
-    # manually check metric memory location and inheritance chain
-    # print(compiled_model.metrics)
-    # for m in compiled_model.metrics:
-    #     if hasattr(m.__class__, '__name__'):
-    #         print(m.name, m.__class__.__name__, m.__class__.__mro__)
-    # input("post-compile - press enter")
-
     model_checkpoint_callback = ModelCheckpoint(Path(model_dir, 'model.hdf5').as_posix(),
                                                 monitor='loss', verbose=1, save_best_only=True)
-    tensorboard_callback = TensorBoard(log_dir=logs_dir.as_posix(), batch_size=batch_size, write_graph=True,
-                                       write_grads=False, write_images=True, update_freq='epoch')
+    # profile_batch = 0 is needed until insufficinet privileges issue resolved with CUPTI
+    #   (_https://github.com/tensorflow/tensorflow/issues/35860)
+    tensorboard_callback = TensorBoard(log_dir=logs_dir.as_posix(), write_graph=True,
```

Comment:
@rak5216 my memory was we were going to pull all tensorboard stuff out? is that right? so we don't have to worry about any of the cupti stuff

Reply:
@Josh-Joseph wasn't sure since it's not causing issues there currently.  I was going to try to see the effect of `profile_batch=0`, but haven't yet.

Reply:
okay sounds good, let's just keep this in then